code/modules/cargo/blackmarket/blackmarket_items/misc.dm

```diff
@@ -0,0 +1,53 @@
+/datum/blackmarket_item/misc
+	category = "Miscellaneous"
+
+/datum/blackmarket_item/misc/cap_gun
+	name = "Cap Gun"
+	desc = "Prank your friends with this harmless gun! Harmlessness guranteed,"
+	item = /obj/item/toy/gun
+	
+	price_min = 50
+	price_max = 200
+	stock_max = 6
+	availability_prob = 80
+
+/datum/blackmarket_item/misc/shoulder_holster
+	name = "Shoulder holster"
+	desc = "Yeehaw, hardboiled friends! This holster is the first step in your dream of becoming a detective and being allowed to shoot real guns!"
+	item = /obj/item/storage/belt/holster
+
+	price_min = 400
+	price_max = 800
+	stock_max = 8
+	availability_prob = 60
+
+/datum/blackmarket_item/misc/holywater
+	name = "Flask of holy water"
+	desc = "Father Lootius' own brand of ready-made holy water."
+	item = /obj/item/reagent_containers/food/drinks/bottle/holywater
```

Comment:
have you considered giving this a chance to be unholy water instead, or would that be a bit too unbalanced if put in a chem grenade or something?

Reply:
Yeah, that sounds fun. sure